crates/matrix-sdk/src/sliding_sync/room.rs

```diff
@@ -208,14 +168,12 @@ struct SlidingSyncRoomInner {
     /// The room ID.
     room_id: OwnedRoomId,
 
-    /// The room representation as a `SlidingSync`'s response from the server.
-    ///
-    /// We update this response when an update is needed.
-    inner: RwLock<v4::SlidingSyncRoom>,
-
     /// Internal state of `Self`.
     state: RwLock<SlidingSyncRoomState>,
 
+    /// The token for back-pagination.
+    prev_batch: RwLock<Option<String>>,
```

Comment:
There's actually a field in `RoomInfo` for this as well: https://github.com/matrix-org/matrix-rust-sdk/blob/ff89315b6e92fe0d8d6c7f4cb53e6aa75d1a914b/crates/matrix-sdk-base/src/rooms/normal.rs#L777-L778.

If the field in `RoomInfo` makes sense is debatable, but I did want to let you know about it.

Reply:
Hmm interesting. I'll update this PR to use the `Roominfo::last_prev_batch` field instead. It seems more relevant. Thanks!

Reply:
Feel free to move this to a separate PR, there has been talk that this should not be part of the `RoomInfo`, though I can't remember the specific complaint anymore.

Reply:
Done here https://github.com/matrix-org/matrix-rust-sdk/pull/3092.